bower.json

```diff
@@ -21,6 +21,8 @@
     "angular-animate": "^1.5.3",
     "angular-mocks": "^1.5.3",
     "angular-patternfly-sass": "3.5.1",
+    "angular-ui-router": "latest",
+    "angular-route": "^1.5.3",
```

Comment:
What's the point of using both `angular-route` and `angular-ui-router`? Probably a mistake..